lua/core/clock.lua

```diff
@@ -198,8 +198,7 @@ function clock.add_params()
     function(x)
       clock.set_source(x)
       if x==4 then
-        crow.input[1].change = function() end
-        crow.input[1].mode("change",2,0.1,"rising")
+        norns.crow.clock_enable()
```

Comment:
is it necessary to also disable clock on crow when the source changes from crow to something else?

Reply:
i'm looking at the code rn and i'm at a loss how it actually works?! the `norns.crow.clock_enable` was just a tiny abstraction to put that crow code into the crow file. i'm worried the routing of the crow event into the clock system was based on some magic behind the scenes that has been broken here.

@artfwo do you remember where the crow 'change' event hooks into the clock system?

Reply:
> @artfwo do you remember where the crow 'change' event hooks into the clock system?

i think the clock handle callback is triggered here:
https://github.com/monome/norns/blob/main/matron/src/device/device_crow.c#L84

Reply:
i'm not sure how this worked previously, but regarding this code:

https://github.com/monome/norns/blob/main/matron/src/clocks/clock_crow.c#L67

does this call only update the global clock if source is set to crow clock reference?

trying to figure out if the crow needs to conditionally call the clock sync based on which clock is enabled--- which is better i think than telling crow to stop sending ^^change which is a generic function and could be connected to other uses

Reply:
my reading of the clock code is that it's fine as-is... that all clock sources are independent (?)